packages/contact-center/station-login/src/helper.ts

```diff
@@ -28,8 +30,10 @@ export const useStationLogin = (props: UseStationLoginProps) => {
     cc.stationLogout({logoutReason: 'User requested logout'})
       .then((res: StationLogoutSuccess) => {
         setLogoutSuccess(res);
-        logoutCb();
-      }).catch((error: any) => {
+        if(logoutCb){
+          logoutCb();
+        }
+      }).catch((error: Error) => {
         console.error(error);
```

Comment:
As of now, we are using console.log and console.error but are we planning to have some some sort of logger module implemented?